### ISSUE TITLE
Google Assistant support?

### ISSUE THREAD

**Issue by mattbruman** [UbhiTS/ad-alexadoorwindowannounce#3]
Is there a way to modify this to work with google assistant? I don't want to create automatons for all my windows and doors, this seems easier.

**Comment by UbhiTS** [comment]
yes, it's on my agenda but I haven't been able to get to it. I know there are requests for the same from others as well. give me some time and I'll have it supported soon ... sorry for the delay !
